LayoutTests/fast/viewport/ios/constant-width-viewport-with-no-scale-after-changing-view-scale.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true ShouldIgnoreMetaViewport=true contentMode=desktop ] -->
+<html>
+<head>
+    <meta name="viewport" content="">
```

Comment:
```suggestion
```

It looks like we could just remove this? Or alternately, we could make this:


```suggestion
    <meta name="viewport" content="width=390">
```

...and delete the code below to dynamically adjust the meta content.